From 98358549dfe94f5813cc090397865bf7c41011cd Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Wed, 20 Dec 2017 15:32:38 +0900
Subject: [PATCH 616/909] arm64: dts: r8a77965-salvator-xs: Add MMNGRBUF driver
 support

commit 389692407490a4e409df84d0210d210f06f444ea from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds MMNGRBUF driver support for the R8A77965 SoC
on the Salvator-X 2nd version board.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77965-salvator-xs.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77965-salvator-xs.dts b/arch/arm64/boot/dts/renesas/r8a77965-salvator-xs.dts
index 3051692c1b9f..c77a4d6acad4 100644
--- a/arch/arm64/boot/dts/renesas/r8a77965-salvator-xs.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77965-salvator-xs.dts
@@ -57,6 +57,10 @@
 		compatible = "renesas,mmngr";
 		memory-region = <&mmp_reserved>, <&lossy_decompress>;
 	};
+
+	mmngrbuf {
+		compatible = "renesas,mmngrbuf";
+	};
 };
 
 &du {
-- 
2.17.1

